MAINTAINERS.md

```diff
@@ -13,11 +13,9 @@ This is the list of maintainers, including their email address for direct commun
 | Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
 | Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Iroha 2 architect                                                             |
```

Comment:
Is Alexandr-san still using his Soramitsu email?

Reply:
Not at this time, unfortunately. We couldn't reach an agreement with Otake-san. 

But I'm still very much interested in the health of the project, 

You could add my current email to it: 

ap886@cantab.ac.uk